far/editor.cpp

```diff
@@ -3800,6 +3800,10 @@ bool Editor::Search(bool Next)
 					break;
 				case KEY_CTRLENTER:
 				case KEY_RCTRLENTER:
+					// MZK 2018-04-01:	After the Find-All dialog is hidden and the cursor is positioned to the found string,
+					//					the next pressed key is still handled by the Find-All dialog, which becomes visible again.
+					// Improvement:		This behavior is probably expected, but some kind of visual indication would be very welcome.
+					// BUG?				Also, navigation keys (arrows, PgUp/PgDn, etc.) show the dialog without the shadow.
```

Comment:
I am going to describe current behavior (modulo minor glitches) on the `@FindAllMenu` help page. Main question is whether Find-All menu should disappear on CtrlEnter? After all, it's a modal dialog. Currently it disappears.

Reply:
It should not.
Ctrl-Enter changes the position without closing the dialog.
Please do not describe this in help.

The current behaviour is probably caused by some of the recent changes in window manager.
@zg0 , could you have a look please?

Reply:
This issue has been fixed, please drop the comment.

Reply:
Thank you. And thank you, @zg0 for the very quick fix. I reverted the comment. The behavior is simple and logical now. I'll describe it with my pleasure.